cli/cmd/compliance_azure.go

```diff
@@ -199,6 +218,24 @@ func init() {
 	complianceAzureGetReportCmd.Flags().StringVar(&compCmdState.Type, "type", "CIS",
 		"report type to display, supported types: CIS, SOC, or PCI",
 	)
+
+	complianceAzureGetReportCmd.Flags().StringSliceVar(&compCmdState.Category, "category", []string{},
+		"filter the compliance report details view by category (identity-and-access-management, s3, logging...)",
```

Comment:
Why is `s3` a category of Azure?
```suggestion
		"filter report details by category (identity-and-access-management, s3, logging...)",
```